scripts/cc-cli/src/utils/balance.ts

```diff
@@ -1,17 +1,36 @@
 import { BN } from "creditcoin-js";
 
-const MICROUNITS_PER_CTC = new BN("1000000000000000000");
+export const MICROUNITS_PER_CTC = new BN("1000000000000000000");
 
 export function parseCTCString(amount: string): BN {
-  return new BN(amount).mul(MICROUNITS_PER_CTC);
+  const split = amount.split(".");
+  const CTC = split[0];
+  const ctcInMicrounits = new BN(CTC).mul(MICROUNITS_PER_CTC);
```

Comment:
Why not use `parseFloat() * MICROUNITS_PER_CTC` and round up to the nearest whole number ?